assets/js/react/components/homepage/advancedSearch/DocumentSearch.jsx

```diff
@@ -167,6 +171,34 @@ class DocumentSearch extends React.Component {
       panelExpanded,
     } = this.state;
 
+    // Hardcode from DB - TODO: get them dynamically
+    const allDocumentTypes = [
+      {
+        id: '1',
+        name: 'Collection',
+      },
+      {
+        id: '4',
+        name: 'Image',
+      },
+      {
+        id: '12',
+        name: 'Text',
+      },
+      {
+        id: '14',
+        name: 'TopographicData',
+      },
+      {
+        id: '17',
+        name: 'Issue',
+      },
+      {
+        id: '18',
+        name: 'Article',
+      },
+    ];
+
```

Comment:
I have hardcoded the document type for the moment because these don't change regularly. In the future, we will need to retrieve them from the database.